src/FixedDataTable.js

```diff
@@ -1003,27 +947,25 @@ var HorizontalScrollbar = React.createClass({
   },
 
   render() /*object*/ {
-    var outerContainerStyle = {
+    const {
+      offset,
+      size,
+    } = this.props;
+
+    const outerContainerStyle = {
       height: Scrollbar.SIZE,
-      width: this.props.size,
+      width: size,
     };
-    var innerContainerStyle = {
+    const innerContainerStyle = {
       height: Scrollbar.SIZE,
-      position: 'absolute',
       overflow: 'hidden',
-      width: this.props.size,
+      width: size,
+      top: offset,
     };
-    FixedDataTableTranslateDOMPosition(
```

Comment:
Rather than using translate3d I switched to offset top since it greatly simplified logic since the translate3d was written w/ an offset from the bottom rather than the top.  It should be trivial to switch back to translate3d written as an offset top and get the performance benefits, so I'll investigate that in a follow up.